src/components/alerting.tsx

```diff
@@ -2408,10 +2013,20 @@ const PollerPages = () => {
       pollers[alertsKey] = poller;
       poller();
     } else {
-      dispatch(alertingErrored('alerts', new Error('prometheusBaseURL not set')));
+      dispatch(alertingErrored('alerts', new Error('prometheusBaseURL not set'), perspective));
     }
-    return () => _.each(pollerTimeouts, clearTimeout);
-  }, [alertsSource, dispatch]);
+    return (): void => {
+      _.each(pollerTimeouts, clearTimeout);
+    };
+  }, [alertsSource, dispatch, perspective, rulesKey, alertsKey, namespace]);
+
+  if (isDev) {
+    return (
+      <Switch>
+        <Route path="/dev-monitoring/ns/:ns/(alerts)" exact component={AlertsPage} />
```

Comment:
I think we don't need the capture group as is a single option
```suggestion
        <Route path="/dev-monitoring/ns/:ns/alerts" exact component={AlertsPage} />
```

Reply:
Remove the capture group in the most recent commit